drivers/storage/portworx/deployment.go

```diff
@@ -323,7 +324,8 @@ func (p *portworx) GetStoragePodSpec(
 		podSpec.DNSPolicy = v1.DNSPolicy(cluster.Annotations[pxutil.AnnotationDNSPolicy])
 	}
 
-	if pxutil.IsCSIEnabled(t.cluster) {
+	pxVersion := pxutil.GetPortworxVersion(cluster)
```

Comment:
We don't need to calculate this again. The version is already present in `t.pxVersion`. We can use `t.pxVersion.LessThan(...)` on the next line.